src/components/flow/exit/__snapshots__/Exit.test.tsx.snap

```diff
@@ -145,46 +145,72 @@ exports[`ExitComp shows missing localization 1`] = `
 </body>
 `;
 
-exports[`ExitComp shows recent messages on mouse over 1`] = `
+exports[`ExitComp shows recent contacts on mouse over 1`] = `
 <body>
   <div>
     <div
-      id="activity_recent_messages"
+      id="activity_recent_contacts"
     >
       <div>
         <div
-          class="recent_messages"
+          class="recent_contacts"
           style="position: absolute; left: 0px; top: 0px;"
         >
           <div
             class="title"
           >
-            Recent Messages
+            Recent Contacts
           </div>
           <div
-            class="message"
+            class="row"
           >
             <div
-              class="text"
+              class="contact"
             >
+              <a
+                href="undefinedeaea6433-d423-41dd-94a9-075de20f322d"
```

Comment:
We should probably give our tests some placeholder endpoints so these look more right.